nx.json

```diff
@@ -164,7 +179,10 @@
         "{projectRoot}/.storybook/**/*",
         "{projectRoot}/tsconfig.storybook.json"
       ],
-      "cache": true
+      "cache": true,
+      "metadata": {
+        "technologies": ["storybook"]
+      }
```

Comment:
Can you remove these? We'll add these separately to the plugins

Reply:
removed